.github/actions/do_build_ock/action.yml

```diff
@@ -44,11 +50,23 @@ inputs:
     description: 'External compiler directories, default is to empty string, which means no external directories'
     default: ""
   hal_description:
-    description: 'Description to be used for HAL, typically risc-v ISA (default RV64GCV)'
-    default: "RV64GCV"
+    description: 'Description to be used for HAL, typically risc-v ISA (default "")'
+    default: ""
+  refsi_tutorial_hal_dir:
```

Comment:
I feel refsi_tutorial are too bespoke to a job and should be removed (and done via the extra args). Same goes for oneapi_con_kit_dir.
Note sure about hal_refsi_thread_mode, maybe that is okay.

Reply:
I'd say keep `hal_refsi_thread_mode` as its used in multiple jobs with different non-default values.
I'll update for the other two you mention.
Ta.

Reply:
to be clear there are two refsi_tutorial ones and one con_kit one (all related to refsi tutorial).

Reply:
These 3 inputs are now deleted.